.github/workflows/handle-stale-contributor-branches.yml

```diff
@@ -0,0 +1,28 @@
+name: Handle stale contributor branches
+on:
+  schedule:
+    # runs every day at midnight (UTC time).
+    - cron:  '0 0 * * *'
+jobs:
+  handle_contributor_stale_branches:
+    runs-on: ubuntu-latest
+    if: github.repository == 'demisto/content'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+      - name: Install Python Dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install pipenv
+      - name: Sync Base Branch to Master
+        env:
+          CONTENTBOT_GH_ADMIN_TOKEN: ${{ secrets.CONTENTBOT_GH_ADMIN_TOKEN }}
+        run: |
+          echo "Handling stale contribution base branches (contrib/*)"
```

Comment:
As I said in a previous comment, you'll need to switch to the containing directory.